docs/nuget-org/nuget-org-faq.yml

```diff
@@ -228,11 +228,17 @@ sections:
           
       - question: How do I change my NuGet.org account username?
         answer: |
-          You cannot. As a matter of policy we do not allow the change of usernames. Also, doing so is a breaking change for users that may have defined [package trust policies based on the package owner](../consume-packages/installing-signed-packages.md#trust-package-owners). The only way to change your username is to create a new account with the desired username. We recommend you delete your existing account before you create a new one, otherwise you will not be able to reuse your registered Microsoft account.
+          You can request a username change by sending an email to [support@nuget.org](mailto:support@nuget.org) from the email address that is attached to the account you want to update.
+          Be sure to include the old username and the new username you would like to change to.
+          We will then review your request and, upon approval, get confirmation from you that we are about the take the correct action and that you understand the consequences.
+          Once you have confirmed, we will change your username.
+
           > [!Important]
-          > Deleting the user will still **reserve** the `username`. You will not be able to reuse the same username again and **this includes the change of casings**. As an example if you created a user with username `mycoolname` and you want to change this to `MyCoolName`(casing changes), it will not be possible after deleting the user.
-          
-          Follow the steps given in [delete your NuGet.org account](#how-to-delete-my-nuget-org-account) section and to [register a new account](individual-accounts.md) with correct username.
+          > The old username will still be **reserved**. You will not be able to reuse the old username again and **this includes the change of casings**.
+          > Any links to your old username profile page (e.g. https://www.nuget.org/profiles/OldUsername) will not be redirected to your new profile.
```

Comment:
These all sound like a section of "consequences" or limitations. We should provide a header for them or a new question and answer? i.e. What happens if I change my NuGet.org account username?

Reply:
I think they are best in proximity with the direct action. (IE linked under how do I change my username). As part of our handling of these requests, we also send the user a full list of all of this before we undertake any action. As such, I'm not sure it merits its own section. But I can make the change if you think it is better organization.